src/react-components/src/__tests__/components/EditionCard-test.tsx

```diff
@@ -4,40 +4,45 @@ import * as Enzyme from "enzyme";
 import * as React from "react";
 
 import EditionCard from "../../components/02-molecules/Cards/EditionCard";
+import BasicLink from "../../components/01-atoms/Links/BasicLink";
+import bem from "../../utils/bem";
+import IconLink from "../../components/01-atoms/Links/IconLink";
 
 describe("EditionCard", () => {
   let regularEditionCard = <EditionCard
     id="card#1"
     blockName=""
     coverUrl="https://placeimg.com/300/400/arch"
     editionHeadingElement={<a href="edition-link" >2004 Edition</a>}
-    editionInfo={["Published in New York by Random House", "Written in English", "Under Creative Commons License"]}
-    readOnlineLink="#readOnlineUrl"
-    downloadLink="#downloadUrl" />;
+    editionInfo={[<>Published in New York by Random House</>, <>Written in English</>, <>Under Creative Commons License</>]}
```

Comment:
Are these tags supposed to have content in them?

Reply:
these are fragments, which are valid.

Reply:
ahhh, TIL!